app/assets/stylesheets/views/_covid.scss

```diff
@@ -141,6 +141,10 @@
     @include govuk-media-query($from: tablet) {
       margin-bottom: govuk-spacing(3);
     }
+
+    .covid__list-item-description--featured {
```

Comment:
I think this class should be `covid__list-item-description` as it's an element, not a modifier on an existing element.